### PR TITLE
Fix ModelBuilder build

### PR DESCRIPTION
**Pull request by ThomasRaoux**

### CODE REVIEW

File: experimental/ModelBuilder/MemRefUtils.h [hunk 1]
```diff
@@ -124,6 +113,18 @@ makeStridedMemRefDescriptor(void *ptr, void *alignedPtr,
   return descriptor;
 }
 
+// Mallocs a StridedMemRefDescriptor<T, N>* that matches the MLIR ABI.
```
Comment (GMNGeoffrey) [inline experimental/ModelBuilder/MemRefUtils.h:116]: Wait how does moving this fix it?

**Comment by GMNGeoffrey** [comment]
>  if it builds (no CI coverage here, right?)

Yeah it would be nice to list in the description how you tested this (as there is no bot coverage)

**Comment by ThomasRaoux** [comment]
I verified that it builds and run on OSS

File: experimental/ModelBuilder/MemRefUtils.h (hunk 1 above)
Comment (ThomasRaoux) [reply experimental/ModelBuilder/MemRefUtils.h:116]: Moved it after the function it calls and added the missing template parameters.

**Pull request merged**